parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -290,14 +290,23 @@ configureCNIIPTables() {
 
 {{if NeedsContainerd}}
 ensureContainerd() {
+  {{if TeleportEnabled}}
+  ensureTeleportd
+  {{end}}
   wait_for_file 1200 1 /etc/systemd/system/containerd.service.d/exec_start.conf || exit $ERR_FILE_WATCH_TIMEOUT
   wait_for_file 1200 1 /etc/containerd/config.toml || exit $ERR_FILE_WATCH_TIMEOUT
   wait_for_file 1200 1 /etc/sysctl.d/11-containerd.conf || exit $ERR_FILE_WATCH_TIMEOUT
   retrycmd_if_failure 120 5 25 sysctl --system || exit $ERR_SYSCTL_RELOAD
-
   systemctl is-active --quiet docker && (systemctl_disable 20 30 120 docker || exit $ERR_SYSTEMD_DOCKER_STOP_FAIL)
   systemctlEnableAndStart containerd || exit $ERR_SYSTEMCTL_START_FAIL
 }
+{{if TeleportEnabled}}
+ensureTeleportd() {
+    wait_for_file 1200 1 /etc/systemd/system/teleportd.service || exit $ERR_FILE_WATCH_TIMEOUT
+    retrycmd_if_failure 120 5 25 systemctl daemon-reload || exit $ERR_SYSTEMCTL_DAEMON_RELOAD
```

Comment:
this is included in systemctlEnableAndStart

Reply:
thanks - using systemctlEnableAndStart